server/services/messaging/message-base.js

```diff
@@ -4,6 +4,7 @@ class MessageBase {
   constructor (name, config, credentials) {
     this.name = name
     this.sbClient = credentials ? ServiceBusClient.createFromAadTokenCredentials(config.host, credentials) : ServiceBusClient.createFromConnectionString(`Endpoint=sb://${config.host}/;SharedAccessKeyName=${config.username};SharedAccessKey=${config.password}`)
+    this.queueClient = this.sbClient.createQueueClient(config.address)
```

Comment:
Both sender and receiver use the `queueClient` so figured it would be better to create it in the base.